src/Facade/Services/SernosNoteService.cs

```diff
@@ -0,0 +1,120 @@
+﻿namespace Linn.Products.Facade.Services
+{
+    using System.Collections.Generic;
+
+    using Linn.Common.Domain.Exceptions;
+    using Linn.Common.Facade;
+    using Linn.Common.Persistence;
+    using Linn.Products.Domain.Linnapps;
+    using Linn.Products.Resources;
+    
+    public class SernosNoteService : ISernosNoteService
+    {
+        private readonly IRepository<SernosNote, int> repository;
+        private readonly ITransactionManager transactionManager;
+
+        public SernosNoteService(
+            IRepository<SernosNote, int> repository, 
+            ITransactionManager transactionManager)
+        {
+            this.repository = repository;
+            this.transactionManager = transactionManager;
+        }
+
+        public IResult<SernosNote> GetSernosNoteById(int sernosNoteId)
+        {
+            var sernosNote = this.repository.FindById(sernosNoteId);
+            if (sernosNote == null)
+            {
+                return new NotFoundResult<SernosNote>();
+            }
+
+            return new SuccessResult<SernosNote>(sernosNote);
+        }
+
+        public IResult<SernosNote> GetSernosNote(string sernosGroup, int? sernosNumber, string transCode)
+        {
+            if (!this.ValidateQuery(sernosGroup, sernosNumber, transCode))
```

Comment:
The fluent validation stuff catches this at the module / end point so you wouldn't need to check here :)